workers/unity/Assets/Gdk/Core/Systems/SpatialOSSendSystem.cs

```diff
@@ -48,7 +48,7 @@ public void RegisterCustomReplicationSystem(ComponentType type)
         {
             if (!registeredReplicators.Contains(type.TypeIndex))
             {
-                Debug.LogWarningFormat(Warnings.CustomReplicationSystemAlreadyExists, type);
+                view.ErrorDispatcher.LogErrorFormat(Warnings.CustomReplicationSystemAlreadyExists, type);
```

Comment:
do we want to change that to an error?

Reply:
I think it should be considered as an error, you shouldn't ever be registering the same system multiple times.

Reply:
This could just be made into try register and return a bool.

Reply:
Then log as a warning.

Reply:
Yeah, this doesn't need to be an error. You shouldn't register a system multiple times, but it won't break anything